src/python/pants/backend/codegen/protobuf/python/gen_rules.py

```diff
@@ -0,0 +1,67 @@
+# coding=utf-8
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (
+    absolute_import,
+    division,
+    generators,
+    nested_scopes,
+    print_function,
+    unicode_literals,
+    with_statement,
+)
+
+import os
+
+from pants.backend.codegen.protobuf.subsystems.protoc import Protoc
+from pants.base.build_root import BuildRoot
+from pants.core.util_rules.distdir import DistDir
+from pants.core.util_rules.external_tool import DownloadedExternalTool, ExternalToolRequest
+from pants.engine.console import Console
+from pants.engine.fs import DirectoryToMaterialize, Workspace
+from pants.engine.goal import Goal, GoalSubsystem, LineOriented
+from pants.engine.platform import Platform
+from pants.engine.rules import goal_rule, subsystem_rule
+from pants.engine.selectors import Get
+
+
+class DummyGetProtocOptions(LineOriented, GoalSubsystem):
+    """Create a runnable binary."""
+
+    name = "dummy-get-protoc"
+
+
+class DummyGetProtoc(Goal):
+    subsystem_cls = DummyGetProtocOptions
+
+
+@goal_rule
+async def dummy_copy_protoc_to_dist(
```

Comment:
Maybe this should just be in a test for now?

Reply:
Yeah, I'm going to just nuke this, it was for a PoC, but I don't think it's needed any more.